### ISSUE TITLE
Some sections don't show entries

### ISSUE THREAD

**Issue by DenyEs** [verbb/hyper#63]
### Describe the bug

Some sections withing Hyper shows emtpy when I am trying to select an entry. Both sections are channels and both sections have multiple entries yet one shows entries in Hyper and the other wont.

<img width="855" alt="Screenshot 2023-05-25 at 09 44 50" src="https://github.com/verbb/hyper/assets/18517642/16bad7d3-a657-41d3-92a1-7a68e83fe414">

<img width="1428" alt="Screenshot 2023-05-25 at 09 45 13" src="https://github.com/verbb/hyper/assets/18517642/7b15aa6e-f892-47e0-b4c4-2ca9d7fe63d4">

There are no errors in console or network tab at all so I am not sure what's happening.

### Steps to reproduce

I am not quite sure to be fair, it seems random and there are no errors.

### Craft CMS version

4.4.13

### Plugin version

1.1.6

### Multi-site?

No

**Comment by engram-design** [comment]
That does seem a little odd. Things to check are:
- This is just a section right? It's not a custom "source"?
- Are there any special permissions for the currently logged-in user?

Are there any settings that are different for "Course Study Mode"? You also mention its random, so it's not happening consistently each time?

**Comment by DenyEs** [comment]
Yes it's a section not a custom source.

Apologies for the confusion, when I say random I mean it happens on a random section (meaning 2 sections that are both channels - one shows entries and the other doesn't).

I think I found out why and it's kind of my fault. Those entries I am trying to link to does not have 'Entry URI format' set in section settings yet, I am rebuilding a site in Craft 4 and I didn't get to it yet.

I tried to add the 'Entry URI format' and it instantly appeared in a list.

**Comment by DenyEs** [comment]
Ever since Categories and Globals moved under 'Entries' it got kind of confusing and cluttered in the sidebar. I suppose I am just going to have to tick sections I want clients to choose from when I am creating my Hyper field.

**Comment by engram-design** [comment]
Ah, yes that'll do it. You can't link to an element that doesn't have a link. I can certainly look at improving that by checking the sections before outputting them.

**Comment by DenyEs** [comment]
Yeah of course :D it was my fault to be fair as I didn't add entry URI format yet and it cofused me as to why it does't show anything. 

If the check can be done on plugins side that would be awesome! I think it save a lot of time for people that use it regulary. Rather than ticking individual fields we'd know that the options it gave are options we can actually link to :)

**Comment by engram-design** [comment]
Updated in [3.0.0-beta.1](https://github.com/verbb/hyper/releases/tag/3.0.0-beta.1).
